src/main/routes/routes.ts

```diff
@@ -6,6 +6,7 @@ export default function(app: Application): void {
 
   app.get('/', app.locals.container.cradle.homeController.get);
   app.get('/search-option', app.locals.container.cradle.searchOptionController.get);
+  app.get('/list', app.locals.container.cradle.hearingListController.get);
```

Comment:
the path for this page is still `list` do we match it to `hearing-list`?

Reply:
Good spot - updated to hearing-list